backend/api/parse_pdf.py

```
@@ -841,7 +1297,7 @@ async def parse_pdf_with_progress_route(
     
     # Start the actual parsing in the background
     import asyncio
-    asyncio.create_task(parse_pdf_with_progress(file, context_files, save_to_db, session_id, db))
+    asyncio.create_task(parse_pdf_with_progress(file, context_files, save_to_db, session_id, db, current_user))
```

Ruff (0.14.3)

Store a reference to the return value of `asyncio.create_task`

(RUF006)